src/main/java/ReadManager.java

```diff
@@ -0,0 +1,25 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadManager {
+
+    private final ArrayList<String> list = new ArrayList<>();
```

Comment:
Please use interface as data type.
Can you use a more meaningful name instead of a `list`?

Reply:
fixed